File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_22232.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request
import os
import importlib
import sys


async def BenchmarkTest22232(request: Request):
    env_value = os.environ.get('USER_INPUT', '')
    data = '%s' % (env_value,)
    sys.path.insert(0, str(data))
    importlib.import_module('report_renderer')
    return {"updated": True}

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest22232.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest22232 {

    private static String normalize(String v) { return v.strip(); }

    @PostMapping(path="/BenchmarkTest22232", consumes="text/plain")
    public void BenchmarkTest22232(@RequestBody String rawBody, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String rawData = rawBody != null ? rawBody : "";
        String data = normalize(rawData);
        Runtime.getRuntime().exec(new String[]{"sh", "-c", "echo " + data}).waitFor(5, java.util.concurrent.TimeUnit.SECONDS);
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
